vars.tf

```diff
@@ -44,6 +44,11 @@ variable "rds_security_group_ids" {
   default     = []
 }
 
+variable "ca_cert_identifier" {
+  description = "Specifies the identifier of the CA certificate for the DB instance"
+  default     = "rds-ca-2019"
```

Comment:
Is this the new value or the old one?

Reply:
New old is 'rds-ca-2015'

Would you prefer the default be the old or new? What is your thinking here?

Reply:
Would we prefer teams to set this if not using the new version or have to set it if using the new version? Setting old as the default means no changes to the teams at this time. When they are updated the teams will need to have the new version or it will show a change. When all changed it is just all use the default, so remove from teams.

Reply:
After discussion - happy with the new value as being the default.